frontend/packages/kubevirt-plugin/src/components/vms/vm.tsx

```diff
@@ -172,10 +174,16 @@ export const VirtualMachinesPage: React.FC<VirtualMachinesPageProps> = (props) =
   ];
 
   const flatten = ({ vms }) => getLoadedData(vms, []);
+  const { namespaced } = VirtualMachineModel;
+  const usedNamespace = !namespace && namespaced ? _.get(match, 'params.ns') : namespace;
```

Comment:
on the second look, this check should not be needed. I think we can use the `namespace` directly.

Reply:
fixed @suomiy